Sources/TensorFlow/Loss.swift

```diff
@@ -36,6 +36,18 @@ public func meanAbsoluteError<Scalar: TensorFlowFloatingPoint>(
     return abs(expected - predicted).mean()
 }
 
+/// Computes the hinge loss between predictions and expectations.
+///
+/// - Parameters:
+///   - predicted: Predicted outputs from a neural network.
+///   - expected: Expected values, i.e. targets, that correspond to the correct output.
+@differentiable(wrt: predicted)
+public func hinge<Scalar: TensorFlowFloatingPoint>(
```

Comment:
I'd suggest renaming this to `hingeLoss(predicted:expected:)`, since "hinge loss" is the canonical name and would be more clear at the call site.

Reply:
Got it, thanks for your feedback!